documentation/overlay.html

```diff
@@ -86,19 +103,33 @@ <h2><a name="mysql">File Structure</a></h2>
 
         <p>
             The texts that are used in the administration panel (as well as some other texts) are taken from various
-            translation files under the <tt>src/i18n</tt> directory. To customize such a file, a file by the same name
-            can be added in the <tt>custom/i18n</tt> directory. The build process will merge both files, which you to
+            translation files under the <code>src/i18n</code> directory. To customize such a file, a file by the same name
+            can be added in the <code>custom/i18n</code> directory. The build process will merge both files, which you to
             add your own properties, as well as override existing properties, without the need to copy all properties
             that aren't of interest to you.
         </p>
 
+    </section>
+
+    <section id="build">
+
         <h2>Build</h2>
-        <p>Simply build the application as normal.  You may need to run the ant <tt>clean</tt>
+
+        <p>
+            Simply build the application as normal. You may need to run the ant <code>clean</code>
```

Comment:
Fix old ant reference. Are ant and maven comparable for 'clean'?

```suggestion
            Simply build the application as normal. You may need to run the <code>./mvnw clean</code>
```

Reply:
Accidentally, yes.